tests/data/test_sql/moz-fx-data-test-project/test/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Test Dataset
+description: |-
+  Use for testing.
+dataset_base_acl: view
+user_facing: true
+labels: {}
+default_table_workgroup_access:
+- role: roles/bigquery.metadataViewer
```

Comment:
nit: you'd generally expect the roles to be reversed for `default_table_workgroup_access` and `workgroup_access`.